backend/fms_core/example_data/csv/container_rename.csv

```diff
@@ -5,8 +5,8 @@ Naming Rules,,,,
 ,,,,
 ,,,,
 #,Old Container Barcode,New Container Barcode,New Container Name,Update Comment
-1,box001,box0001,original_box_2,added 0
-2,tube001,tube002,sample1_tube_2,swapped
-3,tube002,tube001,sample2_tube_2,swapped
+1,box001,box0001b,original_box_2,added 0
```

Comment:
I do not seen a need to change box0001 to box0001b

Reply:
It's a rename.. it could be anything like 'box342423', the only thing is that it must be a unique barcode

Reply:
well it's a test that is supposed to add a 0 and now it adds a b too...

Reply:
right, didnt notice it. lol, maybe we should make it more visible? from box001 to box0010 ?

Reply:
yeah ok ... that would actually help ;)